### PR TITLE
Integration spec views:blog-app

### PR DESCRIPTION
**Pull request by Shubh-Dev**
UPDATE!

- New test add(redirect to show page)
- readme update

Hello Dear reviewer,
integration tests written for Blog-app and n+1 problem solved
Let me know your views.

### CODE REVIEW

File: spec/views/users/user_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
```
Comment (brytebee) [inline spec/views/users/user_index_spec.rb:2]: - [ ] **Required:** Some required tests are missing:

- It should have a test to check that the user can redirect to the posts they made.

**Review by brytebee — COMMENTED** [review]
Hi, @Shubh-Dev👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

- Update README with the [**RUN TESTS** section.](https://github.com/microverseinc/readme-template) This should contain instructions on how one may run the tests in this project. This makes your project more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/views/users/user_index_spec.rb (hunk 1 above)
Comment (rloterh) [reply spec/views/users/user_index_spec.rb:2]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Shubh,  

Your project is complete, great job! There is nothing else to say other than... it's time to merge it 🍾🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- N + 1 is properly addressed
- Integration test with Rspec properly covered
- ReadMe is good

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**